### ISSUE TITLE
Cannot find module 'tslib'

### ISSUE THREAD

**Issue by ddeboer** [rubensworks/rdf-terms.js#47]
When running an app which depends on Comunica, I’m getting:

```
Error: Cannot find module 'tslib'
Require stack:
- /app/node_modules/rdf-terms/index.js
- /app/node_modules/rdf-isomorphic/lib/RdfIsomorphic.js
- /app/node_modules/rdf-isomorphic/index.js
- /app/node_modules/sparqlalgebrajs/lib/sparql.js
- /app/node_modules/sparqlalgebrajs/index.js
- /app/node_modules/@comunica/bus-query-operation/lib/Bindings.js
- /app/node_modules/@comunica/bus-query-operation/lib/ActorQueryOperation.js
- /app/node_modules/@comunica/bus-query-operation/lib/index.js
- /app/node_modules/@comunica/actor-init-query/lib/QueryEngineBase.js
- /app/node_modules/@comunica/actor-init-query/lib/ActorInitQuery.js
- /app/node_modules/@comunica/actor-init-query/lib/index.js
- /app/node_modules/@comunica/query-sparql-rdfjs/lib/QueryEngine.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/app/node_modules/rdf-terms/index.js:3:17)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
```

when starting a production build of the app.

**Comment by rubensworks** [comment]
Ah, this must be because of a broken release from this morning.
Released 1.8.2 that should fix this.
Thanks for reporting!

**Comment by ddeboer** [comment]
Thanks @rubensworks for the quick fix! This resolved our issue.
